src/routes/resources/resources.handlers.ts

```
@@ -1082,3 +1201,391 @@ export const upvoteJobBatch = async (c: Context) => {
+
+    const wasBookmarked = isBookmarked.length > 0;
+
+    if (!wasBookmarked) {
+      // Add bookmark
+      await db.insert(bookmarks).values({
+        resourceId,
+        userId: userLoggedIn.id,
+      });
+      logger.info(
+        `Added bookmark for resource ${resourceId} by user ${userLoggedIn.id}`
+      );
```

ESLint

[error] Missing trailing comma.

(style/comma-dangle)